odoo/orm/fields_properties.py

```diff
@@ -518,6 +518,9 @@ def _parse_json_types(cls, values_list, env, res_ids_per_model):
                     if id_ in res_ids_per_model[res_model]
                 ] if res_model in env else []
 
+            elif property_type == 'html':
+                property_value = html_normalize(html_sanitize(property_value))
```

Comment:
Is the `html_normalize` necessary here?

Reply:
I took the same logic as standard HTML field for consistency